site/helm/helm-integration.md

```diff
@@ -44,8 +44,7 @@ Flux-Helm Integration implementation consists of two parts:
     chartGitPath: mongodb
     releaseName: mongo-database
     values:
-      - name: image
-        value: bitnami/mongodb:3.7.1-r1
+      image: bitnami/mongodb:3.7.1-r1
```

Comment:
It might be useful to add an example of a nested parameter and an example of a parameter with a values in a list format.

Reply:
@aaron7 
Thanks for the added examples.
Can you please also add a case where the value of a parameter, whether nested or not is a list? Thanks.